framework/src/utils/MathFVUtils.C

```diff
@@ -44,9 +51,9 @@ gradUDotNormal(const T &
 #endif
 }
 
+template ADReal gradUDotNormal(
+    const ADReal &, const ADReal &, const FaceInfo &, const MooseVariableFV<Real> &, bool);
```

Comment:
```suggestion
template ADReal
gradUDotNormal(const ADReal &, const ADReal &, const FaceInfo &, const MooseVariableFV<Real> &, bool);
```

Reply:
seems like this was probably clang format

Reply:
When I apply clang-format it splits the lines like that. Precheck would complain if I kept your change.

Reply:
umm well we have 
template XXX
blabla 

everywhere else so let s do:
```suggestion
template ADReal
gradUDotNormal(const ADReal &,
                              const ADReal &,
                              const FaceInfo &,
                              const MooseVariableFV<Real> &,
                              const bool);
```

Reply:
also why not const bool?